subprojects/dependency-management/src/main/java/org/gradle/api/internal/artifacts/ivyservice/resolutionstrategy/DefaultResolutionStrategy.java

```diff
@@ -111,6 +114,8 @@ public DefaultResolutionStrategy(DependencySubstitutionRules globalDependencySub
         this.componentSelectorConverter = componentSelectorConverter;
         this.dependencyLockingProvider = dependencyLockingProvider;
         this.capabilitiesResolution = capabilitiesResolution;
+        this.objectFactory = objectFactory;
+        this.useGlobalDependencySubstitutionRules = objectFactory.property(Boolean.class).convention(true);
```

Comment:
I wasn't sure if there is anything I need to do as the property might be used during _configuration time_. But it seems that's no longer necessary - #18722 (?)